packages/radix-ui-themes/src/helpers/breakpoints.ts

```diff
@@ -154,7 +154,7 @@ function getBaseClassName(
 
 interface GetResponsiveCustomPropertiesOptions {
   value: Responsive<StringOrValue<string>> | Responsive<string> | undefined;
-  customProperty: `--${string}`;
+  customProperty: `--${string}` | `--${string}`[];
```

Comment:
This is needed for props like `px` which need to set both `padding-left` and `padding-right`. The type signature is a bit weird because I needed something that would work with as few changes as possible, but I want to rework it in the next PR that would refactor classnames/styles to be driven by prop defs.